pypxe/nbd/nbd.py

```diff
@@ -84,47 +83,47 @@ def handshake(self, conn, addr):
             [op] = struct.unpack("!I", conn.recv(4))
             if op != 1:
                 # NBD_REP_ERR_UNSUP
-                self.sendreply(conn, addr, 2**31+1, '')
+                self.send_reply(conn, addr, 2 ** 31 + 1, '')
 
-        [namelen] = struct.unpack("!I", conn.recv(4))
+        [namelen] = struct.unpack('!I', conn.recv(4))
         name = conn.recv(namelen)
         if name != self.bd:
             conn.close()
             return 1
 
-        self.logger.debug('Received request for %s from %s', name, addr)
+        self.logger.debug('Received request for {0} from {1}'.format(name, addr))
 
         # size of export
         exportinfo = struct.pack('!Q', self.bdsize)
         flags = (0 if self.write else 2) # readonly?
         exportinfo += struct.pack('!H', flags)
-        exportinfo += "\x00"*(0 if (cflags&2) else 124)
+        exportinfo += '\x00' * (0 if (cflags & 2) else 124)
```

Comment:
Should we use `chr(0)` here like we have previously? I think I remember we switched to `chr(0)` because it returned essentially the same thing (a NULL byte).